### ISSUE TITLE
plotAbunSummary() not working

### ISSUE THREAD

**Issue by OriolAbril** [rjfarmer/mesaplot#3]
Trying to call plotAbunSummary() raises the error
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "build/bdist.linux-x86_64/egg/mesaPlot/__init__.py", line 2143, in plotAbunSummary
ValueError: need more than 0 values to unpack
```
which comes from calling self.\_listAbun(m.hist) at line 2143 of \_\_init\_\_.py, because the prefix is left to be the default value empty string, which for profile files works but not for history files and therefore it returns an empty list. 

plotCenterAbun() does work because it assumes the prefix to be "center\_". The two functions are not much different, so the best solution would probably be a single function plotAbunSummary() with the prefix as one of the inputs (setting a default to "center\_", "surface\_", "total\_mass\_"...). #

**Comment by rjfarmer** [comment]
thanks, clearly haven't run that function for a while.

I'll look into merging the functions, might do some other backend cleaning and merge the other ones like plotAbun and plotBurn* (functions that plot multiple lines) into wrappers around a generic _plotMultiLine function.

**Comment by rjfarmer** [comment]
There is a temp fix in  2c63641 to get things going again while i do a more extensive rework, plotAbunSummary will now take a prefix string so you use the different center_, surface_ versions etc

**Comment by rjfarmer** [comment]
So i pushed lot of changes in 19b3200 to merge duplicated code and now most functions only call either _plotMultiHist or _plotMultiProf. I wouldn't rush to get it though as there are probably alot of breakages with options not be passed properly, as alot of functions have gain options that similar functions had but i hadn't setup yet.
